libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset.component.spec.ts

```diff
@@ -428,6 +428,34 @@ describe('Vertical tabset component', () => {
     flush();
   }));
 
+  it('should focus the active tab when the tab container is opened on mobile view', fakeAsync(() => {
+    mediaQueryController.setBreakpoint('xs');
+    const fixture = createTestComponent();
+    fixture.detectChanges();
+
+    const el = fixture.nativeElement;
+
+    // click show tabs button to open tab group
+    let showTabsButton = el.querySelector('.sky-vertical-tabset-show-tabs-btn');
+    showTabsButton.click();
+    fixture.detectChanges();
+
+    // click second tab in second group
+    clickGroupButton(fixture, 1);
+    clickTab(fixture, 1, 1);
+
+    // open tab group
+    showTabsButton = el.querySelector('.sky-vertical-tabset-show-tabs-btn');
+    showTabsButton.click();
+    fixture.detectChanges();
+
+    const tabsContainer = getTabsContainer(fixture);
+
+    SkyAppTestUtility.fireDomEvent(tabsContainer, 'focus');
```

Comment:
Why are we focusing the tabs container here?

Reply:
good call, i tweaked the unit test with a tick that also means i dont need the flush call anymore